kit/inputs/src/UIMaterialTextView/useAutogrow.ts

```diff
@@ -1,88 +1,21 @@
-import * as React from 'react';
-import {
-    TextInput,
-    NativeSyntheticEvent,
-    TextInputChangeEventData,
-    StyleSheet,
-    Platform,
-} from 'react-native';
-
-import { OnHeightChange, useAutogrowTextView } from '../useAutogrowTextView';
+import type * as React from 'react';
+import type { TextInput } from 'react-native';
+import type { OnHeightChange } from '../useAutogrowTextView';
 import type { UIMaterialTextViewProps } from './types';
 
 export function useAutogrow(
-    ref: React.Ref<TextInput>,
+    _ref: React.Ref<TextInput>,
     onContentSizeChangeProp: UIMaterialTextViewProps['onContentSizeChange'],
     onChangeProp: UIMaterialTextViewProps['onChange'],
-    multiline: UIMaterialTextViewProps['multiline'],
+    _multiline: UIMaterialTextViewProps['multiline'],
     numberOfLinesProp: UIMaterialTextViewProps['numberOfLines'],
-    onHeightChange?: OnHeightChange,
+    _onHeightChange?: OnHeightChange,
```

Comment:
I guess this is used in so many places, hence it needs to be fixed differently...